pyk/src/tests/unit/utils.py

```diff
@@ -16,3 +18,19 @@
 f, g, h = map(KLabel, ('f', 'g', 'h'))
 
 k = KLabel('<k>')
+
+
+def lt(var: str, n: int) -> KApply:
+    return mlEqualsTrue(ltInt(KVariable(var), intToken(n)))
+
+
+def ge(var: str, n: int) -> KApply:
+    return mlEqualsTrue(geInt(KVariable(var), intToken(n)))
+
+
+def config(var: str) -> KApply:
+    return KApply('<top>', KVariable(var))
+
+
+def config_int(n: int) -> KApply:
+    return KApply('<top>', intToken(n))
```

Comment:
I think these should be renamed to make them more specific. `lt` should be `lt_ml_true` and `ge` should be something like `ge_ml_true`.

Reply:
Also, do we need `config` and `config_int`, could we instead use `k` that's already defined above for all your examples? I think it would work, you can just import the variables already used here.

Where you have:

```
config('X') => k(x)
```

And where you have:

```
config_int(3) => k(intToken(3))
```

Then we don't need more helpers.

Reply:
Thank you! I didn't notice https://github.com/runtimeverification/k/blob/42e03545c319847e7df84e6fbc196b2f51c148a0/pyk/src/pyk/kast/inner.py#L123-L125

Reply:
Finished!